docs/Dialects/SV/RationaleSV.md

```diff
@@ -141,6 +141,21 @@ sv.verbatim "MACRO({{0}}, {{1}} reg={{4}}, {{3}})"
             {symRefs = [@reg1, @Module1, @instance1]}
 ```
 
+Substitions also allow format specifier after a ':'. The meaning of said options
+depends on the of the operand or symbol. So far, the follow format specifiers
+are supported:
+
+- Symbol refering to a `hw.hierpath`: the separation character for joining names
+in the path. Defaults to '.'.
```

Comment:
As written in `ExportVerilog`, I think this is a "separation string". If this is intended, then the documentation should talk about it as such as opposed to a separation "character". (Also, is the separation string behavior intended?)